src/Build/parse/generate.py

```diff
@@ -0,0 +1,83 @@
+import os
+from parser import parse_config 
```

Comment:
Should be `from .parser import parse_config` so it doesn't interfere with python's native `parser` module.

Reply:
I have changed the name to parse instead of parser, since it was giving me red mark under the name